src/pages/Dashboard/Dashboard.js

```diff
@@ -13,32 +14,36 @@ import HolofuelLedgerQuery from 'graphql/HolofuelLedgerQuery.gql'
 import { presentHolofuelAmount, POLLING_INTERVAL_GENERAL } from 'utils'
 import './Dashboard.module.css'
 
+// Temporary. This should be removed once we move to hosted release, as we will always show Hosting Card
+const SHOW_HOSTING_CARD = true
+
 export default function Dashboard () {
   // nb: we only call settings here to track hpos connection status (see apolloClient.js for use)
   useQuery(HposSettingsQuery)
   const { data: { hostingReport = {} } = {} } = useQuery(HostingReportQuery)
   const { data: { earningsReport = {} } = {} } = useQuery(EarningsReportQuery)
   const { data: { holofuelLedger: { balance } = { balance: 0 } } = {} } = useQuery(HolofuelLedgerQuery, { fetchPolicy: 'cache-and-network', pollInterval: POLLING_INTERVAL_GENERAL })
```

Comment:
When there is an error returned from this query, the balance displays as 0.  Let's have it default to an empty object, and in the event of an error, return '--' instead.

![image](https://user-images.githubusercontent.com/30459942/88444111-10b73800-cde1-11ea-8174-9bfe1a40dd09.png)

Reply:
I realize that this isn't directly related to the ticket, but has entirely to do with the api calls on hp-admin dashboard page and is a pretty simple lift.  I'll let you make the call on whether to include this here or not.

Reply:
This is now resolved.